src/components/RightSidebar/SharedItems/SharedItemsTab.vue

```diff
@@ -164,13 +165,17 @@ export default {
 		hasSharedItems() {
 			return Object.keys(this.$store.getters.sharedItems(this.token)).length > 0
 		},
+
+		isOneToOne() {
```

Comment:
Now, if we switch to 1-1 conversation, overview will be loaded regardless the tab is open or not.
Maybe watch not after `isOneToOne`, but after `this.$store.getters.getSidebarStatus` to check three items:
 - sidebar is open
 - tab is active
 - token has changed

when it could be the same logic as we have in chatIdentifier in MessageList:
```js
		chatIdentifier() {
			return this.token + ':' + this.active + ':' + this.isSidebarOpen
		},

		watch:
			chatIdentifier() {
				if (this.token && this.active && this.isSidebarOpen) {
					this.getSharedItemsOverview()
				}
			}
		},
```